### ISSUE TITLE
Update project on CurseForge and Modrinth

### ISSUE THREAD

**Issue by megaraichu** [LuckPerms/LuckPerms#3763]
### Description

When attempting to stand up a new server on Aternos for Fabric 1.20.2, Aternos only has access to the versions of LuckPerms that is available on CurseForge and Modrinth.  The latest currently available version for Fabric on CurseForge and Modrinth (5.4.102) crashes on startup for Fabric 1.20.2
Aternos does not let people upload their own mods and people who are using Aternos are at the mercy of what is uploaded to CurseForge and Modrinth.  Please update LuckPerms on CurseForge and Modrinth so Aternos users can use it on the latest version!

### Proposed Behaviour

Updating the project on CurseForge and Modrinth after each update would ensure that everyone has access to the latest versions when using free services like Aternos.

### Extra Details
EDIT:
If at all possible, as Turbotailz proposed, I would suggest adding in some kind of automation to upload the newest version to Modrinth and other platforms with an API.

**Comment by KurtThiemann** [comment]
If it's intended that new versions are only posted on luckperms.net and Modrinth/Curseforge are no longer updated, we can fetch versions directly from luckperms.net (which I think we already do for some platforms).
This would, however, mean that some relevant information, like the supported Minecraft versions for a given LuckPerms version, could no longer be displayed on Aternos, as this information currently seems to be hardcoded in a JavaScript bundle instead of being available through the API.

**Comment by megaraichu** [comment]
I do not believe Aternos is willing to do that, but I can send a request to have them pull directly from the LuckPerms website.  I don't see them going for that though.

But if it is decided that the projects on Modrinth and CurseForge are no longer going to be updated, then so be it, and I will need to find a new hosting option.  It would be a shame though, as Aternos is one of the better free hosting options out there.

Is there any possibility that a decision can be made on this in any kind of definite timeframe?

**Comment by Turbotailz** [comment]
Luck is just a very busy guy and doesn't have time to send updates to the many platforms LP is hosted.

Probably would be worth setting up some sort of automation to upload versions to Modrinth (and other platforms with an API) as part of the build process in Github Actions.

**Comment by megaraichu** [comment]
If that is possible, I'd like to make that my suggestion.  I'll add it to the OP

**Comment by lucko** [comment]
I'm usually quite quick to upload. The upgrade to 1.20.2 for Forge is proving to be a bit of a pain, which is why I haven't published them yet.

There is already some automation for uploading: https://github.com/lucko/mod-publish :)

**Comment by Laarryy** [comment]
Closing this as it's no longer an issue.
